build.fsx

```diff
@@ -163,6 +164,15 @@ Target "RunTests" (fun _ ->
             XmlOutputPath = Some "TestResults.xml" })
 )
 
+// --------------------------------------------------------------------------------------
```

Comment:
nunit instead of xunit does make this a little messier to handle, if this was xunit it would of just been picked up by RunTests